internal/callbacker/store/postgresql/migrations/000002_add_quarantine_ts.up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE callbacker.callbacks ADD COLUMN quarantine_until TIMESTAMPTZ;
```

Comment:
I think correct english would be `quarantined_until`, but not sure. 
Thinking of naming I think the callback itself is not quarantined, but rather the URL. 

What would you think of `postponed_until`?
IMO that would be a bit more expressive name of the field

Reply:
Agree, it's a better name.
Changed